CanHazFunny/CanHazFunny/ConsoleOutputWriter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CanHazFunny
```

Comment:
Error 0161 is requesting that file scoped namespace is used rather than block scoped({})

```suggestion
namespace CanHazFunny;
```
deleting the curly braces and using a semi colon instead for every namespace should fix the 0161 errors throughout